src/model.ts

```diff
@@ -466,7 +468,7 @@ export class JSORMBase {
         let attributeName = key
 
         if (this.klass.camelizeKeys) {
```

Comment:
Since we're allowing backwards-incompatible changes in 1.0, I'd like to remove this and support `letterCase === "camelized"` the same way you already have `letterCase === "dasherized"`

Reply:
In my opinion they both have a different semantic: 

letterCase allows to set the format of the api. 
camelizeKeys tells jsorm if it should camelize the keys that come from the api or if it should keep the format as it is.

Reply:
I think this really comes down for naming. Effectively we now have `deserializeKey` and `serializeKey` methods - anything dealing with casing should be within these two methods.

So one option would be to move `if (this.camelizeKeys)` into the `serializeKey` method's logic. But it doesn't really make sense to have one thing called `letterCase` and another thing called `camelizeKeys`.

What about something like `static keyFormat = { from: 'dasherized', to: 'camelized' }`?

Reply:
yes putting it to an object with from and to makes totally sense. I think keyFomat is to general what's about keyCase?

Reply:
`keyCase` sounds good to me 👍  Please make sure Typescript is only allowing from/to strings we have whitelisted. @wadetandy suggests `'dash' | 'camel' | 'snake'`